src/scales/scale.category.js

```diff
@@ -63,17 +63,21 @@ module.exports = Scale.extend({
 		return me.ticks[index - me.minIndex];
 	},
 
-	// Used to get data value locations.  Value can either be an index or a numerical value
-	getPixelForValue: function(value, index, datasetIndex) {
+	_getParams: function() {
```

Comment:
Could we set these values in `_configure` so that they don't need to be recomputed for every data point?

Reply:
done